src/style/components/list/conversation-list-calling-cell.less

```diff
@@ -87,12 +87,12 @@
       background-color: var(--green-500) !important;
 
       &:hover {
-        border-color: var(--green-600) !important;
-        background-color: var(--green-600) !important;
+        border-color: @green-600 !important;
```

Comment:
just curious why some places this is less style vars and some CSS style? any particular reason for the split